fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1941,13 +1941,18 @@ queryNoWith
     : queryPrimary (ORDER BY sortItem (',' sortItem)*)? (limitElement)?
     ;
 
-temporalClause
+queryPeriod
```

Comment:
why change this?

Reply:
The version which we newly added is not temporalClause